lib/utilities.js

```diff
@@ -14,6 +14,14 @@ const safe = fn => (...args) => {
   }
 };
 
+const captureMaxStack = () => {
+  const oldLimit = Error.stackTraceLimit;
+  Error.stackTraceLimit = Infinity;
+  const stack = new Error().stack;
```

Comment:
Maybe we should exclude `captureMaxStack` line from actual `stack`?

Reply:
I don't this should deal with that, it would require processing the stack and also may be confusing to the user.
IMHO, this should be done as a separate function if needed.